### ISSUE TITLE
lr_scheduler

### ISSUE THREAD

**Issue by gsdgui** [Lightning-AI/pytorch-lightning#17951]
### Bug description

The learning rate scheduler is not working properly. Is there something wrong with my settings?

![image](https://github.com/Lightning-AI/lightning/assets/58936734/a16811b6-bc35-4b25-bbfb-4192a6585292)
![image](https://github.com/Lightning-AI/lightning/assets/58936734/1ab37a65-d4f6-4d83-8095-c04e1fefc98f)
![image](https://github.com/Lightning-AI/lightning/assets/58936734/0949ea42-7d7e-48d2-8400-0ce0976a45ef)

### What version are you seeing the problem on?

master

### Error messages and logs

```
# Error messages and logs here please
```

### Environment

<details>
  <summary>Current environment</summary>

```
#- Lightning Component (e.g. Trainer, LightningModule, LightningApp, LightningWork, LightningFlow):
#- PyTorch Lightning Version (e.g., 1.5.0):
#- Lightning App Version (e.g., 0.5.2):
#- PyTorch Version (e.g., 2.0):
#- Python version (e.g., 3.9):
#- OS (e.g., Linux):
#- CUDA/cuDNN version:
#- GPU models and configuration:
#- How you installed Lightning(`conda`, `pip`, source):
#- Running environment of LightningApp (e.g. local, cloud):
```

</details>

**Comment by awaelchli** [comment]
Closing, because a repro is missing and there is no clear indication that it is an issue with Lightning. If you still face this issue, please open a new bug report with a code sample that shows the problem (no screenshots please). 
Thanks
